Makefile

```diff
@@ -98,46 +108,51 @@ calypso-dev:
 	@cd $(CALYPSO_DIR) && CALYPSO_ENV=$(CALYPSO_ENV) npm run -s start
 
 # Build desktop bundle
-build-desktop:
+build-desktop: rebuild-deps
+	@echo "Building Desktop..."
 ifeq ($(NODE_ENV),development)
 	@echo "$(CYAN)$(CHECKMARK) Starting Desktop Server...$(RESET)"
 endif
 
 	NODE_PATH=calypso$/server$(ENV_PATH_SEP)calypso$/client CALYPSO_SERVER=true npx webpack --config $(THIS_DIR)$/webpack.config.js
 
-	@echo "$(CYAN)$(CHECKMARK) Desktop built$(RESET)"
+	@echo "$(GREEN)$(CHECKMARK) Desktop built$(RESET)"
 
 # Package App
 package:
+	@echo "Packaging app..."
+
 	@npx electron-builder build -$(BUILD_PLATFORM)
 
-	@echo "$(CYAN)$(CHECKMARK) App built$(RESET)"
+	@echo "$(GREEN)$(CHECKMARK) App built$(RESET)"
 
 # Combined steps for building app 
 build: build-source package
 
 # Perform checks
-checks: check-node-version-parity secret
-
+checks: check-version-parity secret
 
 # Check for secret and confirm proper clientid for production release
+SECRETS := ./calypso/config/secrets.json
```

Comment:
the `CALYPSO_DIR` variable should be an absolute reference and therefore should work regardless of where the `Makefile` was invoked. when we move to relative file references we change the links based on the current working directory from whence this was invoked, so if we do `make wp-desktop/` or `make ..` then we'll probably point to something unexpected

Reply:
Yeah - good call. For some reason I was having trouble with earlier on Windows, but it's not an issue any longer. Reverted to absolute paths in 979b8bb.